Userland/Utilities/gev.cpp

```diff
@@ -0,0 +1,295 @@
+/*
+ * Copyright (c) 2021, xSlendiX <gamingxslendix@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCore/ArgsParser.h>
+#include <LibCore/MimeData.h>
+#include <LibGUI/Application.h>
+#include <LibGUI/Painter.h>
+#include <LibGUI/Widget.h>
+#include <LibGUI/Window.h>
+#include <unistd.h>
+
+class EventViewerWidget : public GUI::Widget {
+    C_OBJECT(EventViewerWidget);
+
+public:
+    void set_all_enabled(bool all_enabled) { m_all_enabled = all_enabled; }
+    void set_mouse_enabled(bool mouse_enabled) { m_mouse_enabled = mouse_enabled; }
+    void set_key_enabled(bool key_enabled) { m_key_enabled = key_enabled; }
+    void set_drag_enabled(bool drag_enabled) { m_drag_enabled = drag_enabled; }
+    void set_drop_enabled(bool drop_enabled) { m_drop_enabled = drop_enabled; }
+    void set_resize_enabled(bool resize_enabled) { m_resize_enabled = resize_enabled; }
+    void set_enter_enabled(bool enter_enabled) { m_enter_enabled = enter_enabled; }
+    void set_leave_enabled(bool leave_enabled) { m_leave_enabled = leave_enabled; }
+    void set_theme_enabled(bool theme_enabled) { m_theme_enabled = theme_enabled; }
+    void set_fonts_enabled(bool fonts_enabled) { m_fonts_enabled = fonts_enabled; }
+
+protected:
+    virtual ~EventViewerWidget() override
+    {
+    }
+
+    virtual void paint_event(GUI::PaintEvent& event) override
+    {
+        GUI::Painter painter(*this);
+        painter.add_clip_rect(event.rect());
+
+        painter.fill_rect(rect(), Color::White);
+    }
+    virtual void resize_event(GUI::ResizeEvent& event) override
+    {
+        if (!(m_all_enabled || m_resize_enabled))
```

Comment:
With the above this becomes:

```suggestion
        if (!m_resize_enabled)
```